test/e2e-tests.sh

```diff
@@ -29,9 +29,9 @@ install_pipeline_crd
 
 failed=0
 
-# Run the integration tests
+# Run the integration tests (including the conformance tests)
 header "Running Go e2e tests"
-go_test_e2e -timeout=20m ./test/... || failed=1
+go_test_e2e tags=e2e,conformance -timeout=20m ./test/... || failed=1
```

Comment:
Should we run them seperate though ? (maybe even different jobs in the end)

Reply:
eventually i agree, but in the meantime, probably better to run them than not? (i think there's only 1 anyway)